### DISCUSSION TITLE
技术预览版 - 5

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#272] [通知]
## 更改记录

- 添加常见问题文档 @Richasy (#270)

## 🚀 功能

- 调整时间线页面布局 @Richasy (#268)

## 🐛 Bug 修复

- 修复ARM64启动失败的问题 @Richasy (#271)
- 调整直播线路优先级并在进入直播间时重置长连接 @Richasy (#264)

## 🧰 管理与维护

- 更新构建器 @Richasy (#246)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2109.2301.pre-release'>技术预览版 - 5</a>.</em>

**Comment by X0980** [comment]
一天一更，🐮🍺

**Comment by Stapxs** [comment]
半夜更新辛苦了 😵

**Comment by xiaohanghaijia** [comment]
才下午吧4下载完毕

**Comment by luciaxlx** [comment]
感谢大佬！

**Comment by zxz4514** [comment]
大佬牛啤
